newton/_src/solvers/coupled/proxy_utils.py

```
@@ -0,0 +1,334 @@
+        out_proxy_body_f: Output spatial wrenches on proxy bodies.
+    """
+    contact_id = wp.tid()
+    if contact_id >= rigid_contact_count[0]:
+        return
+
+    body0 = int(contact_body0[contact_id])
+    body1 = int(contact_body1[contact_id])
+    if body0 < 0 or body1 < 0:
+        return
+
+    is_proxy0 = int(0)
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)